lib/redis.rb

```diff
@@ -555,6 +555,10 @@ def migrate(key, options)
   # @param [String, Array<String>] keys
   # @return [Integer] number of keys that were deleted
   def del(*keys)
+    if keys.is_a?(Array) && keys.empty?
+      return 0
+    end
```

Comment:
```suggestion
    return 0 if keys.empty?
```

`keys` can't be anything else than an array.